examples/knative/powercli/kn-pcli-vds-config/README.md

```diff
@@ -0,0 +1,158 @@
+# kn-pcli-vds-config
+Example Knative PowerCLI function for enforcing specific VDS configuration - automatically reset VDS MTU to a desired state
+
+# Step 1 - Build
+
+Create the container image locally to test your function logic.
```

Comment:
nit: please follow NSX Tag Sync example regarding image naming and letting the user know that build is an optional step (they don't need to build if they don't make changes)

Reply:
I think this gets confusing for folks that are first timers to Docker - I followed the example but I added a commented area with the correct local Docker syntax, lmk what you think.